sessions/css-basics/social-card/readme.md

```diff
@@ -0,0 +1,12 @@
+# CSS Basics: Challenges
+
+## Social Card
+
+Your task is to recreate the following social card.
+![social card example](./assets/social-card.png)
+
+- [ ] Link to the styles.css stylesheet in the head of the
+      index.html file.
+- [ ] Style all elements on the page to recreate these social card styles.
+- [ ] Use class selectors to target the correct elements.
+- [ ] Find a fitting font on fonts.google.com and [import it into your project](#styling-fonts).
```

Comment:
The `#styling-fonts` link doesn't link anywhere. Why not add a link to an article?
```suggestion
- [ ] Find a fitting font on fonts.google.com and [import it into your project](https://www.digitalocean.com/community/tutorials/css-using-google-fonts).
```